src/kernel/capinv.c

```diff
@@ -246,6 +336,8 @@ cap_cpy(struct captbl *t, capid_t cap_to, capid_t capin_to, capid_t cap_from, ca
 		type = ctfrom->type;
 		sz   = __captbl_cap2bytes(type);
 
+		/* don't allow cap copy on SCB/DCB */
+		if (type == CAP_SCB || type == CAP_DCB) return -EINVAL;
```

Comment:
Oh, wow, so no delegation at all for these. OK, somewhat surprising. Likely to keep things simple.